### ISSUE TITLE
Cannot use Annotated without generic parameters

### ISSUE THREAD

**Issue by MHDante** [microsoft/pyright#1533]
Consider the following Code:

``` python
from typing import Union
from typing_extensions import Annotated, get_origin

assert get_origin(Union[int, str]) is Union
assert get_origin(Annotated[int, "cheem"]) is Annotated
```

this conforms to the expected usage of get_origin as described here:

https://docs.python.org/3/library/typing.html#typing.get_args

however, pyright shows the following error:

```
PS C:\Repos\pyright-test> pyright .
Loading configuration file at C:\Repos\pyright-test\pyrightconfig.json
Assuming Python platform Windows
stubPath C:\Repos\pyright-test\typings is not a valid directory.
Searching for source files
Found 9 source files
C:\Repos\pyright-test\proj1\src\annotated_test.py
  4:47 - error: Expected one type argument and one or more annotations for "Annotated"
1 error, 0 warnings, 0 infos
Completed in 0.967sec
PS C:\Repos\pyright-test> 
```
![image](https://user-images.githubusercontent.com/4652122/109058194-b18c0f80-76b0-11eb-9c94-d41ee7c52893.png)

Tested with 
pyright 1.1.114
pylance 2021.2.4-pre.1

**Comment by erictraut** [comment]
This will be fixed in the next release.

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.115, which was just published. It will also be in the next release of Pylance.
